firmware/fullhan_fls.ksy

```diff
@@ -0,0 +1,54 @@
+meta:
+  id: fullhan_fls
+  title: Fullhan FLS
+  file-extension: fls
+  license: CC0-1.0
+  endian: le
+  encoding: UTF-8
+doc: |
+  Test files:
```

Comment:
Thanks for declaring where to find test files (I'll check them out), but I think `/doc` should also include some general description of the format. You put the .ksy spec in the `firmware` directory, which suggests that the format is somehow firmware-related, but anything beyond that is left to the reader to guess. I guess it's a firmware update format of IP cameras of a certain brand Fullhan? At least one or two sentences would go a long way for someone not knowing the purpose of the format or perhaps trying to find a .ksy spec for it via certain keywords.